From 60641508cc5ef9a7e8030398d4df85743f6d9285 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 9 Nov 2015 16:23:28 +0800
Subject: [PATCH 0663/5242] MLK-11842 video: epdc: change 'CONFIG_PM_RUNTIME'
 to 'CONFIG_PM'

commit  9ada529f1b0c1bdcf649770469bc74648ff1579f from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'CONFIG_PM_RUNTIME' has been eliminated and all the users
should use 'CONFIG_PM' directly. So do this change for epdc and
epdc-v2.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mxc_epdc_fb.c    |    2 +-
 drivers/video/fbdev/mxc/mxc_epdc_v2_fb.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/fbdev/mxc/mxc_epdc_fb.c b/drivers/video/fbdev/mxc/mxc_epdc_fb.c
index 6061cc7..4ba8e57 100644
--- a/drivers/video/fbdev/mxc/mxc_epdc_fb.c
+++ b/drivers/video/fbdev/mxc/mxc_epdc_fb.c
@@ -5176,7 +5176,7 @@ static int mxc_epdc_fb_resume(struct device *dev)
 #define mxc_epdc_fb_resume	NULL
 #endif
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int mxc_epdc_fb_runtime_suspend(struct device *dev)
 {
 	release_bus_freq(BUS_FREQ_HIGH);
diff --git a/drivers/video/fbdev/mxc/mxc_epdc_v2_fb.c b/drivers/video/fbdev/mxc/mxc_epdc_v2_fb.c
index 0e3e66e..4a1a474 100644
--- a/drivers/video/fbdev/mxc/mxc_epdc_v2_fb.c
+++ b/drivers/video/fbdev/mxc/mxc_epdc_v2_fb.c
@@ -5703,7 +5703,7 @@ static int mxc_epdc_fb_resume(struct device *dev)
 #define mxc_epdc_fb_resume	NULL
 #endif
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int mxc_epdc_fb_runtime_suspend(struct device *dev)
 {
 	release_bus_freq(BUS_FREQ_HIGH);
-- 
1.7.9.5

